.circleci/config.yml

```
@@ -67,9 +66,10 @@ jobs:
 
         steps:
             - checkout
+            - run: echo $PYVERSION > ver.txt
 
             - restore_cache:
-                key: deps1-{{ .Branch }}-{{ checksum "tests/requirements.txt" }}
+                key: deps1-{{ .Branch }}-{{ checksum "package.json" }}-{{ checksum "ver.txt" }}
```

```suggestion
                key: deps1-{{ .Branch }}-{{ checksum "tests/requirements.txt" }}-{{ checksum "ver.txt" }}
```

And `save_cache` needs updating to match